boards/arc/arduino_101_sss/board.cmake

```diff
@@ -7,8 +7,7 @@ endif()
 set(BOARD_DEBUG_RUNNER openocd)
 
 board_runner_args(dfu-util "--pid=8087:0aba" "--alt=sensor_core")
-set(PRE_LOAD targets 1)
-board_runner_args(openocd "--cmd-pre-load=\"${PRE_LOAD}\"" "--gdb-port=3334")
+board_runner_args(openocd --cmd-pre-load "targets 1" "--gdb-port=3334")
```

Comment:
Should we rename this as `board_west_args` ?

Reply:
I know you keep the concept of a runner inside the `west` code, but I am just wondering if the user, when providing a new board, should be made aware that ultimately `CMake` will use `west`.

Reply:
> Should we rename this as board_west_args ?

I can make this change if you really want, but I think it's a bit confusing, since west will support more commands than just flash/debug/debugserver. These arguments are specifically what the build system passes just to those commands (the ones that use the `runner` subpackage), not to all west invocations.

> I know you keep the concept of a runner inside the west code, but I am just wondering if the user, when providing a new board, should be made aware that ultimately CMake will use west.

Good point.

I would suggest we either keep this as-is, and clearly document in the board porting guide that `runner` here means west's runner subpackage (and define clearly what that is and its relationship to the build system), or choose more specific names like:

- `board_west_flash_debug_args`
- `board_west_runner_args`

What do you think?

Reply:
I like `board_west_runner_args`. At the end of the day I do not think it worthwhile to actually list the operations the parameters apply to, but rather just to give the user an idea of to whom these arguments are passed to. If the user really wants to investigate it can then run `west -h` to find out where those parameters belong.

Reply:
No, my comment only concerned the actual name of the CMake macro, not the implications of it. I am happy with `-H` and with the behavior as is.